src/main/java/org/trypticon/pdn/nrbf/arrays/records/ArraySingleString.java

```diff
@@ -56,7 +56,7 @@ public static ArraySingleString readFrom(LittleEndianDataInputStream stream) thr
         ArrayInfo arrayInfo = ArrayInfo.readFrom(stream);
         List<LengthPrefixedString> members = new ArrayList<>(arrayInfo.getLength());
         for (int i = 0; i < arrayInfo.getLength(); i++) {
-            members.add(LengthPrefixedString.readFrom(stream));
+            //members.add(LengthPrefixedString.readFrom(stream));

```

Comment:
I left this one out; I'm not sure what happens to the stream if I skip reading the contained strings from the stream.